### DISCUSSION TITLE
Repeater without password

### DISCUSSION THREAD

**Discussion by virtualized-human** [Mygod/VPNHotspot#489] [General]
This feature please

**Comment by Mygod** [comment]
No plan for this. Feel free to share if you find a way to get around device limit.

**Reply by virtualized-human** [reply]
The repeater function is the way how i get around.

**Reply by virtualized-human** [reply]
But why do you restrict the repeater feature so much? What's the problem to make users able to use other authentication methods?

**Comment by virtualized-human** [comment]
I will pay for adding it / getting a custom version idc xd

**Comment by Mygod** [comment]
Feel free to submit PR.
